docs/Using-Fleet/Detail-Queries-Summary.md

```diff
@@ -13,6 +13,16 @@ Following is a summary of the detail queries hardcoded in Fleet used to populate
 SELECT serial_number, cycle_count, health FROM battery;
 ```
 
+## chromeos_profile_user_info
+
+- Platforms: chrome
+
+- Query:
+
```

Comment:
Would be good to remove the line below "Query" so these code blocks line up with the bullet points, but I see from the comment at the top that this file is auto-generated. Will add this to the notes I'm compiling, just mentioning it now as an FYI.